features/Context/ContentType.php

```diff
@@ -7,17 +7,26 @@
 use Behat\Behat\Context\Context;
 use Behat\Behat\Context\SnippetAcceptingContext;
 use Behat\MinkExtension\Context\RawMinkContext;
+use Exception;
 use eZ\Publish\API\Repository\ContentTypeService;
 use eZ\Publish\API\Repository\Exceptions\NotFoundException;
+use eZ\Publish\API\Repository\Repository;
 use PHPUnit_Framework_Assert as Assertion;
 
 class ContentType extends RawMinkContext implements Context, SnippetAcceptingContext
 {
     /** @var \eZ\Publish\API\Repository\ContentTypeService */
     private $contentTypeService;
 
-    public function __construct(ContentTypeService $contentTypeService)
+    /**
+     * Current content type within this context.
+     * @var \eZ\Publish\API\Repository\Values\ContentType\ContentType
+     */
+    private $currentContentType;
+
+    public function __construct(Repository $repository, ContentTypeService $contentTypeService)
```

Comment:
Strange, as its up to Symfony2Extension's ArgumentResoler to convert service names to actual instances, and seems to work file with services define in the yaml.

ref: https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Bundle/PlatformBehatBundle/Context/Argument/AnnotationArgumentResolver.php#L25